shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -566,11 +575,12 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
         out.write(buf, 0, t);
         t = is.read(buf);
       }
-      if (!tmpDst.renameTo(dstFile)) {
+      if (!tmpDst.renameTo(outputFile)) {
         throw new IOException(
-            "Failed to rename " + tmpDst.getPath() + " to destination " + dstPath);
+            "Failed to rename " + tmpDst.getPath() + " to destination "
+                    + "file://" + outputFile.getPath());
```

Comment:
I don't think we need the `"file://" +` here since `tmpDst.getPath()` won't have it either.